src/aat/resources/features/F-108/F-108.feature

```diff
@@ -0,0 +1,268 @@
+    Then the response [contains the previously created case]
+    And the response has all other details as expected
+
+    @S-621
+  Scenario: Should return case for exact match on a Number field
+      Given a case that has just been created as in [S-621_Create_Case_Private_Autotest1]
+      And logstash has finished indexing case data
+      And a user with [a valid profile]
+      And the request [is configured to search for exact number field value from previously created case]
+      And a request is prepared with appropriate values
+      When it is submitted to call the [external search query] operation of [CCD Data Store Elastic Search API]
+      Then the response [contains the previously created case]
+      And the response has all other details as expected
+
+    @S-622
+  Scenario: Should return case for exact match on a PhoneUK field
+    Given a case that has just been created as in [S-622_Create_Case_Private_Autotest1]
+    And logstash has finished indexing case data
+    And a user with [a valid profile]
+    And the request [is configured to search for exact PhoneUK value from previously created case]
+    And a request is prepared with appropriate values
+    When it is submitted to call the [external search query] operation of [CCD Data Store Elastic Search API]
+    Then the response [contains the previously created case]
+    And the response has all other details as expected
+
+    @S-623
+  Scenario: Should return case for exact match on a Text Area field
+    Given a case that has just been created as in [S-623_Create_Case_Private_Autotest1]
+    And logstash has finished indexing case data
+    And a user with [a valid profile]
+    And the request [is configured to search for exact Text Area field value from previously created case]
+    And a request is prepared with appropriate values
+    When it is submitted to call the [external search query] operation of [CCD Data Store Elastic Search API]
+    Then the response [contains the previously created case]
+    And the response has all other details as expected
+
+
+  @S-624
+  Scenario: Should return case for exact match on a Text field
+    Given a case that has just been created as in [S-624_Create_Case_Private_Autotest1]
+    And logstash has finished indexing case data
+    And a user with [a valid profile]
+    And the request [is configured to search for exact Text field value from previously created case]
+    And a request is prepared with appropriate values
+    When it is submitted to call the [external search query] operation of [CCD Data Store Elastic Search API]
+    Then the response [contains the previously created case]
+    And the response has all other details as expected
+
+
+  @S-625
+  Scenario: Should return case for exact match on a YesNo field
+    Given a case that has just been created as in [S-625_Create_Case_Private_Autotest1]
+    And logstash has finished indexing case data
```

Comment:
Could you please move an equivalent, but more flexible and re-usable form of this sentence to the framework?
I suggest something like:
`And a wait time of [**5**] seconds [**to allow for Logstash to index the case just created**]`

Parts in bold are variable. First part can be a any number including fractions. Second part is to flexibly describe purpose of delay.

Reply:
This needs to be appended to the framework as suggested, instead of keeping a nice domain-agnostic capability in data store api. @tomxelliott